packages/db/src/schema.ts

```diff
@@ -757,6 +758,33 @@ export const apExamReward = pgTable("ap_exam_reward", {
   coursesGranted: json("courses_granted").$type<APCoursesGrantedTree>().notNull(),
 });
 
+export const libraryTraffic = pgTable(
+  "library_traffic",
+  {
+    id: integer("id").primaryKey(),
+    libraryName: varchar("library_name").notNull(),
+    locationName: varchar("location_name").notNull(),
+    trafficCount: integer("traffic_count").notNull(),
+    trafficPercentage: real("traffic_percentage").notNull(),
+    timestamp: timestamp("timestamp").notNull(),
+  },
+  (table) => [index().on(table.locationName)],
```

Comment:
You may also want an index on `libraryName` since queries can be done using that as a filter. Otherwise, such queries will resort to a full table scan (!!!)